src/Currency/AbstractCurrency.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PCF\ValueObject\Currency;
+
+use PCF\ValueObject\ValueObjectInterface;
+
+abstract class AbstractCurrency implements ValueObjectInterface
+{
+    
+    public const FULL_NAME = '';
+    public const SUBUNIT = '';
+    public const ISO_CODE = '';
+    public const SYMBOL = '';
+    
+    protected const SUBUNIT_TO_UNIT = 1;
```

Comment:
every unit have 'subunit_to_unit = 100' what if change this const to 100 and remove everywhere where  we dont need set it custom ?